django/db/backends/base/schema.py

```diff
@@ -858,7 +858,7 @@ def _create_index_name(self, table_name, column_names, suffix=""):
         The name is divided into 3 parts: the table name, the column names,
         and a unique digest and suffix.
         """
-        table_name = strip_quotes(table_name)
+        table_name = split_identifier(table_name)[1]
```

Comment:
I'd use `namespace, table_name` instead of string indexing. (or _ instead of namespace if unused variable names is a problem with the first suggestion).